stories/list/object/object-list.stories.js

```diff
@@ -0,0 +1,798 @@
+   </div>
+   <div class="fd-list__object fd-list__object--bottom-right">
+   <span class="fd-object-status fd-object-status--positive">Second status</span>
+   </div>
+   </div>
+   <div class="fd-list__object fd-list__object--bottom-row">
+   <div class="fd-list__object fd-list__object--bottom-left">
+     <span class="fd-list-attribute">Third Attribute</span>
+   </div>
+   </div>
+   </div>
+   </div>
+  </li>
+</ul>
+</div>
+`;
+
+selection.storyName = 'Selection';
+
+selection.parameters = {
+    docs: {
+        iframeHeight: 305,
+        storyDescription: 'To create selection list use `fd-list--selection ` and `fd-list--row-selection` along with `fd-list and fd-list--object` modifier.'
+    }
+};
+
+export const multiSelection = () => `
+<h4>Object List Item With Multi Selection</h4>
+<div style="max-width: 450px">
+<ul class="fd-list fd-list--object fd-list--selection" role="list">
+  <li role="listitem" tabindex="-1" class="fd-list__item is-selected">
+  <div class="fd-form-item fd-list__form-item">
+          <input type="checkbox" class="fd-checkbox" id="Ai4ez6111Z" checked="" aria-labelledby="O09lk1">
+          <label tabindex="-1" class="fd-checkbox__label" for="Ai4ez6111Z"></label>
+  </div>
+    <div class="fd-list__content">
+    <div class="fd-list-optional">
+    <span class="fd-list-optional__text">Optional inline text</span>
+    </div>
+    <div class="fd-list__object fd-list__object--top-row">
+    <span class="fd-avatar fd-avatar--s" style="background-image: url('http://lorempixel.com/400/400/nature/4/')"></span>
+    <div class="fd-list__object fd-list__object--top-left fd-object-identifier">
+      <p class="fd-object-identifier__title">
+    Fitbit Versa Smart Watch, Black Fitbit Versa Smart Watch
+     </p>
+     </div>
+    <div class="fd-list__object fd-list__object--top-right fd-object-number fd-object-number--positive">
+    <span class="fd-object-number__text">956.00</span>
+    <span class="fd-object-number__unit">Euro</span>
+    </div>
+   </div>
+   <div class="fd-list__object fd-list__object--bottom">
+   <div class="fd-list__object fd-list__object--bottom-row">
+   <div class="fd-list__object fd-list__object--bottom-right">
+      <span class="fd-object-status fd-object-status--positive">First status</span>
+   </div>
+   </div>
+   <div class="fd-list__object fd-list__object--bottom-row">
+   <div class="fd-list__object fd-list__object--bottom-left">
+     <span class="fd-list-attribute">Second Attribute</span>
+   </div>
+   <div class="fd-list__object fd-list__object--bottom-right">
+   <span class="fd-object-status fd-object-status--positive">Second status</span>
+   </div>
+   </div>
+   <div class="fd-list__object fd-list__object--bottom-row">
+   <div class="fd-list__object fd-list__object--bottom-left">
+     <span class="fd-list-attribute">Third Attribute</span>
+   </div>
+   </div>
+   </div>
+   </div> 
+  </li>
+  <li role="listitem" tabindex="-1" class="fd-list__item">
```

Comment:
```suggestion
  <li role="option" aria-selected="true" tabindex="0" class="fd-list__item">
```

Reply:
updated role and tab index values, as it is not a selected row did not update aria-selected.